ocn_020_nrt_rw0_nutrient_mole_concentration/contents/src/__init__.py

```diff
@@ -0,0 +1,600 @@
+    # get a list of all the layers
+    layers = r.json()['data']['attributes']['layer']
+    # create an empty list to store the layer IDs
+    layerIDs =[]
+    # go through each layer and add its ID to the list
+    for layer in layers:
+        # only add layers that have Resource Watch listed as its application
+        if layer['attributes']['application']==['rw']:
+            layerIDs.append(layer['id'])
+    return layerIDs
+
+def flushTileCache(layer_id):
+    """
+    Given the API ID for a GEE layer on Resource Watch,
+    this function will clear the layer cache.
+    If the cache is not cleared, when you view the dataset on Resource Watch, old and new tiles will be mixed together.
+    INPUT   layer_id: Resource Watch API layer ID (string)
+    """
+    # generate the API url for this layer's cache
+    apiUrl = 'http://api.resourcewatch.org/v1/layer/{}/expire-cache'.format(layer_id)
+
+    # create headers to send with the request to clear the cache
+    headers = {
+    'Content-Type': 'application/json',
+    'Authorization': os.getenv('apiToken')
+    }
+
+    # clear the cache for the layer
+    # sometimetimes this fails, so we will try multiple times, if it does
+
+    # specify that we are on the first try
+    try_num=1
+    tries=4
+    while try_num<tries:
+        try:
+            # try to delete the cache
+            logging.info(headers)
+            logging.info(apiUrl)
+            r = requests.delete(url = apiUrl, headers = headers, timeout=1000)
+            # if we get a 200, the cache has been deleted
+            # if we get a 504 (gateway timeout) - the tiles are still being deleted, but it worked
+            if r.ok or r.status_code==504:
+                logging.info('[Cache tiles deleted] for {}: status code {}'.format(layer_id, r.status_code))
+                return r.status_code
+            # if we don't get a 200 or 504:
+            else:
+                # if we are not on our last try, wait 60 seconds and try to clear the cache again
+                if try_num < (tries-1):
+                    logging.info('Cache failed to flush: status code {}'.format(r.status_code))
+                    time.sleep(60)
+                    logging.info('Trying again.')
+                # if we are on our last try, log that the cache flush failed
+                else:
+                    logging.error('Cache failed to flush: status code {}'.format(r.status_code))
+                    logging.error('Aborting.')
+            try_num += 1
+        except Exception as e:
+              logging.error('Failed: {}'.format(e))
+
+'''
+FUNCTIONS FOR THIS DATASET
+
+The functions below have been tailored to this specific dataset.
+They should all be checked because their format likely will need to be changed.
+'''
+def getCollectionName(val,n):
+    '''
+    get GEE collection name
+    INPUT   val: variable to be used in asset name (string)
+            n: index for that variable in the data dictionary (string)
```

Comment:
I think this is a number instead of a string? 
```suggestion
            n: index for that variable within the list of biogeochemical parameters in the data dictionary (integer)
```